common-practices-tools/best-practices-for-managers-and-team-members.md

```diff
@@ -1,32 +1,32 @@
+-   Figure out a balance between what is best for the project and the team member. Look into ways to align them.
 
 ## Manager not on the same project
 
--   Organize a regular 1-1 or slack check-in with the project tech lead (or team tech lead or project manager) to discuss and share feedback about a team member. Try a cadence of at least every two/three months.
--   Ask for feedback from leads, project managers, other team members about a team member's good performance, areas for improvement, and so on. Share those with the team members in a supportive manner.
--   If permitted by the client, find a way to sit in on sprint demos/iteration reviews to see a team member's presentation. Ask as needed, project and product managers about sprint demos/iteration reviews and if they have any specific feedback. Where government background checks are required to work on a project be prepared for permission not to be granted.
--   Look for appreciations for the team member and note it down for future reference. Share/discuss with the team members the appreciation.
--   Discuss project concerns with leads, project manager to figure out ways in which you can support the project or the team member.
+-   Organize a regular 1:1 or slack check-in with the project tech, lead/team tech lead, or project manager to discuss and share feedback about a team member. Try a cadence of at least every two/three months.
+-   Ask for feedback from leads, project managers, and other team members about a member's performance, areas for improvement, and so on. Share those with the team members in a supportive manner.
+-   If the client permits, find a way to attend sprint demos/iteration reviews to see a team member's presentation. Ask, as needed, project and product managers about sprint demos/iteration reviews and if they have any specific feedback. Where government background checks are required to work on a project, be prepared for permission not to be granted.
+-   Look for appreciation for the team member and note it down for future reference. It would help if you shared/discussed this appreciation with the team members.
+-   Discuss project concerns with leads/project managers to figure out ways in which you can support the project or the team member.
 
 ## Team member on the same project
 
--   Discuss project roles with your manager and how project interactions occur (reference project team working agreement).
+-   You should discuss project roles with your manager and how project interactions occur (you can refer to the project team working agreement).
 -   Share concerns about the project and brainstorm ways to resolve them.
--   Don't only focus on the project in 1-1s. Discuss your prodev, goals, energy gains/drains, work-life balance, and so on.
+-   Focus on more than just the project in 1:1s. Discuss your prodev, goals, energy gains/drains, work-life balance, etc.
```

Comment:
@sharjeelaziz could we drop the 'just' as the below suggestion recommends? You should be able to see the warning on the 'Files changed' tab.